### ISSUE TITLE
JOSS review - data to generate figures

### ISSUE THREAD

**Issue by KatyBrown** [Clinical-Genomics-Umea/fraggler#14]
Issue raised as part of the JOSS review process to address the "Data sharing" and "Reproducibility" checks https://github.com/openjournals/joss-reviews/issues/6869?notification_referrer_id=NT_kwDOAIfBYbMxMTAzNjY2ODgzMzo4ODk2ODY1

I think that the data and code required to generate Figure 1 in the manuscript is provided in the "demo" folder but from the manuscript this is not very clear - it would be helpful to state explicitly, e.g. in the figure legend, that the figure uses this data. It would also be good to say where the data comes from - if it's a manually generated example it's no problem, but just for clarity.

I couldn't find the data used for benchmarking and to generate Figure 2. Again it would be good to clearly state where this data is available. Since this is a benchmark against other software, the source of the data also needs to be provided, or a brief protocol if this is locally generated data.

**Comment by willros** [comment]
Hi @KatyBrown!

Thanks for the input. I have now added files and scripts inside the `paper/figures` directory which can be used to generate Figure 2. 

Generating the files using the Peakscanner software demands some manual work and is quite cumbersome (one of the reason we created Fraggler), but I have described how this was done in the `FIGURES.md`. 

Let me know if you want anything else regarding this issue!
William
